var/spack/repos/builtin/packages/sgpp/set_boost_lib_path_internally.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/SConstruct b/SConstruct
```

Comment:
Just in case this diff comes from an existing commit, I like to reference the patch from github, include a checksum and just get it from there, as it gives 1. context to the change and 2. doesn't need to be included here.

It's a bit subjective what's nicer though, so this is also just a suggestion.

Reply:
hmm, this patch actually does not map to one commit on github cleanly (it contains parts from two commits). I basically mostly included it because I noticed that the testing with ```spack install --test=root sgpp@3.4.0``` was broken which was rather annoying.

I like your suggestion though! I think that's how I will handle this the next time a similar issue comes up! However, for this specific patch I would just leave it as it currently is.